packages/dds/task-manager/src/interfaces.ts

```diff
@@ -5,19 +5,126 @@
 
 import { ISharedObject, ISharedObjectEvents } from "@fluidframework/shared-object-base";
 
+/**
+ * Describes the event listener format for {@link ITaskManagerEvents} events.
+ *
+ * @param taskId - The unique identifier of the related task.
+ */
+export type TaskEventListener = (taskId: string) => void;
+
 /**
  * Events emitted by {@link TaskManager}.
  */
 export interface ITaskManagerEvents extends ISharedObjectEvents {
 	/**
-	 * Notifies when the local client has reached the front of the queue, left the queue, or a task was completed.
-	 * Does not account for known pending ops, but instead only reflects the current state.
+	 * Notifies when the specified task has been exclusively assigned to the client.
+	 *
+	 * @remarks Does not account for known pending ops, but instead only reflects the current state.
+	 *
+	 * @eventProperty
+	 */
+	(event: "assigned", listener: TaskEventListener);
```

Comment:
@scottn12 I took a stab at documenting these independently. Would love feedback about specifics. Also any additional details that might be worth adding to each.